backend/src/test/scala/cromwell/backend/wdl/PureStandardLibraryFunctionsSpec.scala

```diff
@@ -27,4 +27,15 @@ class PureStandardLibraryFunctionsSpec extends FlatSpec with Matchers {
     PureStandardLibraryFunctions.transpose(Seq(Success(inArray))) should be(Success(expectedResult))
   }
 
+  behavior of "length"
+
+  it should "get the right answers" in {
```

Comment:
That would seem more obvious, but I assumed the author of the esteemed and similarly situated `transpose` had a reason for choosing this arrangement which I copied.  😛